### PR TITLE
JAMES-3447 Email/set create should return blobId, threadId, size

### PR DESCRIPTION
**Pull request by LanKhuat**

### CODE REVIEW

File: server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ public void setUp() throws MailboxException, DomainListException, UnknownHostExc
         when(outbox.getMailboxPath()).thenReturn(MailboxPath.forUser(USER, OUTBOX));
         
         when(outbox.appendMessage(any(MessageManager.AppendCommand.class), any(MailboxSession.class)))
-            .thenReturn(new MessageManager.AppendResult(new ComposedMessageId(OUTBOX_ID, TestMessageId.of(23), MessageUid.of(1)),
+            .thenReturn(new MessageManager.AppendResult(new ComposedMessageId(OUTBOX_ID, TestMessageId.of(23), MessageUid.of(1)), 1L,
```
Comment (chibenwa) [inline server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java:197]: Name that magic constant

File: server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesUpdateProcessorTest.java [hunk 2]
```diff
@@ -200,7 +200,7 @@ public void setUp() throws MailboxException, DomainListException, UnknownHostExc
 
         when(outbox.appendMessage(any(MessageManager.AppendCommand.class), any(MailboxSession.class)))
             .thenReturn(new MessageManager.AppendResult(
-                new ComposedMessageId(OUTBOX_ID, TestMessageId.of(23), MessageUid.of(1)),
+                new ComposedMessageId(OUTBOX_ID, TestMessageId.of(23), MessageUid.of(1)), 0L,
```
Comment (chibenwa) [inline server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesUpdateProcessorTest.java:203]: Name that magic constant

File: server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java (hunk 1 above)
Comment (Arsnael) [reply server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java:197]: I thought was not always necessary for tests?

Comment (chibenwa) [reply server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessorTest.java:197]: It do not cost too much, IMO let's take the time...

**Review by chibenwa — APPROVED** [review]
But we need a JIRA number.

You can reuse one of Email/set create I think.

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**